api/lib/utils.js

```diff
@@ -125,6 +125,53 @@ function isValidLedgerWord(ledger) {
   return (/^current$|^closed$|^validated$/.test(ledger));
 }
 
+function getFee(remote) {
+  // https://wiki.ripple.com/Transaction_Fee#Base_Fees
+  var FEE_BASE_REFERENCE = 10;  // this is currently a constant in rippled
```

Comment:
I'll add that rippled.cfg link with commit hash. It's probably worth keeping the wiki link too since there's no way to know where it will be migrated to.